llvm/lib/Target/RISCV/RISCVInsertVSETVLI.cpp

```diff
@@ -426,7 +438,8 @@ DemandedFields getDemanded(const MachineInstr &MI, const RISCVSubtarget *ST) {
 /// values of the VL and VTYPE registers after insertion.
 class VSETVLIInfo {
   struct AVLDef {
-    const MachineInstr *DefMI;
+    // Every AVLDef should have a VNInfo.
+    const VNInfo *VNInfo;
```

Comment:
This renaming exists a potential compilation error during the LLVM building. 

gcc version: 8.5.0

```
llvm-project/llvm/lib/Target/RISCV/RISCVInsertVSETVLI.cpp:442:19: error: declaration of ‘const llvm::VNInfo* {anonymous}::VSETVLIInfo::AVLDef::VNInfo’ [-fpermissive]
     const VNInfo *VNInfo;
                   ^~~~~~
In file included from llvm-project/llvm/include/llvm/CodeGen/LiveIntervals.h:25,
                 from llvm-project/llvm/lib/Target/RISCV/RISCVInsertVSETVLI.cpp:31:
llvm-project/llvm/include/llvm/CodeGen/LiveInterval.h:53:9: error: changes meaning of ‘VNInfo’ from ‘class llvm::VNInfo’ [-fpermissive]
   class VNInfo {
         ^~~~~~
```

Reply:
Oh thanks for catching that, I guess it doesn't show up with clang. Will rename!